mozilla_django_oidc_db/forms.py

```diff
@@ -0,0 +1,58 @@
+import json
+
+from django import forms
+from django.utils.translation import gettext_lazy as _
+
+import requests
+
+from .constants import OIDC_MAPPING
+from .models import OpenIDConnectConfig
+
+
+class OpenIDConnectConfigForm(forms.ModelForm):
+    class Meta:
+        model = OpenIDConnectConfig
+        fields = "__all__"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.fields["oidc_op_authorization_endpoint"].required = False
+        self.fields["oidc_op_token_endpoint"].required = False
+        self.fields["oidc_op_user_endpoint"].required = False
+
+    def clean(self):
+        cleaned_data = super().clean()
+
+        discovery_endpoint = cleaned_data.get("oidc_op_discovery_endpoint")
+
+        # Derive the endpoints from the discovery endpoint
+        if discovery_endpoint:
+            try:
+                response = requests.get(discovery_endpoint, timeout=10)
```

Comment:
This should rather be something like:

```py
f"{discovery_endpoint}.well-known/openid-configuration"
```

The `.well-known` path is fixed and can be hardcoded (or defined as a constant). Please also add to the help text that the `.well-known` part will be added automatically.